### PR TITLE
Backwards-compatible io

### PR DESCRIPTION
**Pull request by shamb0t**
This PR adds backwards-compatibility so that old OrbitDB addresses can be read.

Depends on https://github.com/orbitdb/orbit-db-store/pull/43, https://github.com/orbitdb/ipfs-log/pull/234, https://github.com/orbitdb/orbit-db-access-controllers/pull/20

### CODE REVIEW

File: test/utils/mem-store.js [hunk 1]
```diff
@@ -1,95 +1,63 @@
 'use strict'
 
 const multihashing = require('multihashing-async')
-const mh = require('multihashes')
+const CID = require('cids')
+const pify = require('pify')
```
Comment (haadcode) [inline test/utils/mem-store.js]: What is pify and what does it do?

Comment (shamb0t) [reply test/utils/mem-store.js]: Promisifies methods

Comment (haadcode) [reply test/utils/mem-store.js]: Got it :+1:

**Comment by haadcode** [comment]
LGTM :+1:

**Pull request merged**